tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -148,10 +148,8 @@ protected DataReader importData() throws IOException {
                 CHGraph lg = g.getGraph(CHGraph.class, weighting);
                 fillAllowedEdges(lg.getAllEdges(), allowedEdges);
                 printMiscUnitPerfTests(lg, isCH, encoder, count * 100, allowedEdges);
-                printTimeOfRouteQuery(hopper, isCH, isLM, count, "routingCH", vehicleStr, true, -1, "");
-                printTimeOfRouteQuery(hopper, isCH, isLM, count, "routingCH_sod", vehicleStr, true, -1, DIJKSTRA_BI_SOD);
-                printTimeOfRouteQuery(hopper, isCH, isLM, count, "routingCH_no_instr", vehicleStr, false, -1, "");
-                printTimeOfRouteQuery(hopper, isCH, isLM, count, "routingCH_sod_no_instr", vehicleStr, false, -1, DIJKSTRA_BI_SOD);
+                printTimeOfRouteQuery(hopper, isCH, isLM, count, "routingCH", vehicleStr, true, -1);
+                printTimeOfRouteQuery(hopper, isCH, isLM, count, "routingCH_no_instr", vehicleStr, false, -1);
```

Comment:
also here I would test 1. with SOD 2. without SOD and 3. with SOD and no instructions instead of running two queries per measurement